torchbenchmark/__init__.py

```diff
@@ -14,6 +15,31 @@
 model_dir = 'models'
 install_file = 'install.py'
 
+# Enum class to hold all the available domain names
+class Domain(Enum):
+    COMPUTER_VISION = "computer vision"
+    OTHER_COMPUTER_VISION = "other computer vision"
```

Comment:
OTHER_COMPUTER_VISION should be a task not a domain

Reply:
Ok, the score.yml defines this as Domain:

```other computer vision:
        Background_Matting:
 ```

Reply:
https://github.com/pytorch/benchmark/blob/master/torchbenchmark/score/score.yml#L14
i think it is defined as a task there, but since it comes after a commented out block, I wonder if yaml parser did not read it correctly? that is odd..